tests/storage/hotplug.go

```diff
@@ -497,6 +497,7 @@ var _ = SIGDescribe("Hotplug", func() {
 		})
 
 		table.DescribeTable("Should be able to add and use WFFC local storage", func(addVolumeFunc func(name, namespace, volumeName, claimName, bus string), removeVolumeFunc func(name, namespace, volumeName string)) {
+			tests.SkipIfNonRoot(virtClient, "root owned disk.img")
```

Comment:
cdi isn't setting 107 owner on the disk.img?! I'd hope this would work.

Reply:
Unfortunately no. Maybe we can revisit this?